python/py-httpbin/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-httpbin
+version             0.7.0
+revision            0
+
+platforms           darwin
+license             MIT
+maintainers         nomaintainer
+
+description         HTTP Request and Response Service
+long_description    ${description}
+
+homepage            https://github.com/requests/httpbin
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+checksums           sha256  cbb37790c91575f4f15757f42ad41d9f729eb227d5edbe89e4ec175486db8dfa \
+                    rmd160  22b1bb0151212aa71aaa8cb8fb1aec7e1e9c0b5a \
+                    size    92613
+
+python.versions     37
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append \
+                    port:py${python.version}-brotlipy \
+                    port:py${python.version}-decorator \
+                    port:py${python.version}-flask \
+                    port:py${python.version}-itsdangerous \
+                    port:py${python.version}-markupsafe \
+                    port:py${python.version}-raven \
```

Comment:
since the dependency is declared as ```raven[flask]``` (see [here](https://github.com/postmanlabs/httpbin/blob/2607b7e6c78bab21f9791331bdde15c583521c90/setup.py#L38)), you will need additionally a dependency on ```py-blinker``` (see [here](https://github.com/getsentry/raven-python/blob/d7d14f61b7fb425bcb15512f659626648c494f98/setup.py#L42)). This is -at least for now- above the capabilities of ```upt``` (or otherwise our backend) to get these type of special dependency-declarations correct, but we should investigate whether this information is present in the PyPI json data (ping @Korusuke and @Steap)

Reply:
Done. I saw that being declared, but it was too late to keep investigating what raven[flask] was precisely, and it seemed to work without it :)